win32/src/gballoc_hl_metrics.c

```diff
@@ -125,54 +126,60 @@ static void internal_copy_latency_data(GBALLOC_LATENCY_BUCKETS* latency_buckets_
     }
 }
 
-static int wasInitialized = 0;
+static call_once_t g_lazy = 0;

```

Comment:
>0 [](start = 28, length = 1)

LAZY_INIT_NOT_DONE?

Reply:
Of course :)

---
In reply to: [467113867](https://github.com/Azure/azure-c-pal/pull/31#discussion_r467113867) [](ancestors = 467113867)